addons/hr_fleet/models/employee.py

```diff
@@ -13,7 +13,7 @@ class Employee(models.Model):
     def action_open_employee_cars(self):
         self.ensure_one()
         cars = self.env['fleet.vehicle.assignation.log'].search([
-            ('driver_id', 'in', (self.user_id.partner_id | self.sudo().address_home_id).ids)]).mapped('vehicle_id')
+            ('driver_id', 'in', (self.sudo().address_home_id).ids)]).mapped('vehicle_id')
```

Comment:
I'm not sure if drivers are more often `self.user_id.partner_id` or `self.address_home_id` in real use-cases and what are the implication of this change. An employee could have two cars, one linked to the user, one to the private partner

Reply:
Drivers should be the private address normally.

Reply:
@LucasLefevre  If a partner P1 (user_id.partner_id of employee E) is the driver of a car C
When changing P1 to P2 (address_home_id of E) as the driver of C, the same car is counted twice on the employee form view.
From this [ticket](https://www.odoo.com/web#id=2431316&action=333&active_id=49&model=project.task&view_type=form&cids=1&menu_id=4720) , it has been decided to progressively only set the address_home_id as driver.
Perhaps I will make this changement only for master. Are you ok for the other part of the fix?